tests/integration/module-resolution/index.ts

```diff
@@ -11,6 +11,10 @@ const allowedPaths = new Set([
 	'/third-party/slash-create',
 ]);
 
+// Needed by the react 3rd party library
+// Turning on nodejs_compat v2 causes the `slash-create` library to fail
```

Comment:
In theory, should code that runs successfully without `node_compat` also run with `node_compat` or are there scenarios where it's expected to break?

Reply:
I think the problem is to do with code "sniffing" what the platform is. If the library determines that it is running in Node.js (because of the reasonably high-fidelity polyfills provided by nodejs_compat) then it might run different code that doesn't actually work in workerd.

Reply:
I see now that `slash-create` is being imported from `slash-create/web`.

https://github.com/flarelabs-net/vite-plugin-cloudflare/blob/5893df31fedb6a8517cd02da6a714df37fb3be81/tests/integration/module-resolution/src/third-party/slash-create.ts#L1-L6

For Node, the standard import should be used so that explains the error.